### ISSUE TITLE
part 3 feedback

### ISSUE THREAD

**Issue by andreashandel** [Ehsan-suez/EhsanSuez-MADA-project#3]
please updated all documentation (readme files, etc.), remove anything that's not part of the project.

hard to follow the exploratory script, is very poorly formatted. please improve. also, add more commentary as to what you do and why.

i don't see any analysis.

please have a manuscript file that contains text and figures/tables in an easily readable/formatted manner. also a supplement file that has additional results. everything else can be in R/Rmd scripts for the code part.

Overall,  I'm quite concerned with the state of your project as it is. This should be much more robust and comprehensive and further along at this stage. Please make sure for part 4 you catch up and get to a stage that is closer to the comprehensive final project I expect from everyone.

**Comment by Ehsan-suez** [comment]
Thank you for your feedback, Dr. Handel!

I have reorganized the whole project. Could you please have a look and let me know if this seems better? And please let me know if you have any additional suggestions.
